src/store/modules/command_params.js

```diff
@@ -11,6 +11,9 @@ const state = {
     mount_ra: '',
     mount_dec: '',
     mount_object: '',
+    mount_ha: '0',
+    mount_az: '0',
+    mount_alt: '45',
```

Comment:
Any particular reason for these defaults? Especially since that '45' is different than a '30' written below? If so, include in comments

Reply:
facing north, pointing vaguely up. Better than pointing down in the dirt. 30 is as good as 45, I will change it.